testutils/clustermanager/util_test.go

```diff
@@ -53,10 +53,44 @@ func TestGetResourceName(t *testing.T) {
 
 		out, err := getResourceName(ClusterResource)
 		if err != nil {
-			t.Fatalf("getting resource name for cluster, wanted: 'no error', got: '%v'", err)
+			t.Fatalf("getting resource name for cluster, want: 'no error', got: '%v'", err)
 		}
 		if out != data.exp {
-			t.Fatalf("getting resource name for cluster, wanted: '%s', got: '%s'", data.exp, out)
+			t.Fatalf("getting resource name for cluster, want: '%s', got: '%s'", data.exp, out)
+		}
+	}
+}
+
+func TestGetClusterLocation(t *testing.T) {
+	datas := []struct {
+		region, zone string
+		want         string
+	}{
+		{"a", "b", "a-b"},
+		{"a", "", "a"},
+	}
+	for _, data := range datas {
+		if got := getClusterLocation(data.region, data.zone); got != data.want {
+			t.Errorf("getting cluster location with region %q and zone %q, want: %q, got: %q",
```

Comment:
Got before want:
```suggestion
			t.Errorf("Cluster location with region %q and zone %q = %q, want: %q",
```
And swap the args below, obviously ;-)